features.js

```diff
@@ -556,7 +556,7 @@ module.exports = {
     // TODO: edge, mobile platforms?
     browserType: function(client, peerConnectionLog) {
         var peerConnectionConfig = getPeerConnectionConfig(peerConnectionLog);
-        return peerConnectionConfig.browserType || 'unknown';
+        return 'browserType' in peerConnectionConfig ? peerConnectionConfig.browserType : 'unknown';
```

Comment:
Why was this change needed?

Reply:
This is to fix the `undefined property error` as shown here: https://app.datadoghq.com/logs?cols=service&event=AQAAAW_tEXEAqPN0PAAAAABBV190RVhFQXhNaDFWQ2dNTXV0cA&from_ts=1580229807720&index=main&live=true&messageDisplay=inline&query=browserType&stream_sort=desc&to_ts=1580230707720

Reply:
Ah, in that case I believe this will still throw an undefined error since `in` requires that peerConnectionConfig be defined. And the error is showing that peerConnectionConfig is undefined.  You could do something like `return (peerConnectionConfig && peerConnectionConfig.browserType) || 'unknown';`

Reply:
Ah yeah, good catch. Fixed.